docs/pt/developer-docs/game-identification.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Métodos de Identificação de Jogos por Console
+description: Guia detalhado sobre os métodos de hash usados para identificação de jogos em vários consoles de jogos, incluindo Nintendo, Sega, Sony e outros.
+---
+
+# Identificação de Jogos
+
+Esta página detalha o método de hash usado para cada sistema suportado.
+
+## The 3DO Company
+
+- **3DO Interactive Multiplayer** usa um hash personalizado.
+  - O cabeçalho do volume (primeiros 132 bytes do setor 0) e o conteúdo do arquivo LaunchMe são submetidos ao hash.
+
+## Amstrad
+
+- **Amstrad CPC** usa os checksums MD5 de cada disco.
+  - Como o salvamento e outras manipulações podem alterar os dados do disco, cópias locais das imagens carregadas são necessárias para garantir que seus hashes não mudem entre as sessões.
+
+## Apple
+
+- **Apple II** usa os checksums MD5 de cada disco.
+  - Como o salvamento e outras manipulações podem alterar os dados do disco, cópias locais das imagens carregadas são necessárias para garantir que seus hashes não mudem entre as sessões.
+
+## Arcade
+
+- **Arcade** usa um hash personalizado.
+  - A string do nome do arquivo sem a extensão (`path/galaga.zip` -> `galaga`) é submetida ao hash. É sensível a maiúsculas e minúsculas.
+
+## Arduboy Inc.
+
+- **Arduboy** usa um hash personalizado.
+  - Os arquivos .hex do Arduboy contêm texto ASCII. Normalizamos para terminações de linha Unix e depois realizamos um hash do arquivo completo.
+
+## Atari
+
+- **Atari 2600** usa o checksum MD5.
+
+- **Atari 7800** usa o checksum MD5, a menos que a ROM tenha um cabeçalho.
+
+  - Se a ROM começa com `\1ATARI7800`, os primeiros 128 bytes são ignorados e conteúdo restante do arquivo é submetido ao hash. Se a ROM não começa com `\1ATARI7800`, todo o arquivo é submetido ao hash.
+
+- **Atari Jaguar** usa o checksum MD5.
+
+- **Atari Jaguar CD** usa um hash personalizado.
+
+  - Um bloco de cabeçalho está localizado dentro do primeiro setor do disco. Este bloco indica a localização e o tamanho do código de inicialização, que é carregado e submetido ao hash.
+  - Se o hash resultante for `254487b59ab21bc005338e85cbf9fd2f`, é um bootloader comum para jogos homebrew e o código real do jogo está localizado na faixa 2. Se um bloco de cabeçalho separado contendo uma nova localização e tamanho for encontrado na faixa 2, então esses dados são carregados e submetidos ao hash.
+
+- **Atari Lynx** usa o checksum MD5, a menos que a ROM tenha um cabeçalho.
+  - Se a ROM começa com `LYNX\0`, os primeiros 64 bytes são ignorados e o conteúdo restante do arquivo é submetido ao hash. Se a ROM não começa com `LYNX\0`, o arquivo inteiro é submetido ao hash.
+
+## Bandai
+
+- **WonderSwan** usa o checksum MD5.
+
+- **WonderSwan Color** usa o checksum MD5.
+
+## Coleco
+
+- **ColecoVision** usa o checksum MD5.
+
+## Fairchild
+
+- **Channel F** usa o checksum MD5.
+
+## GCE
+
+- **Vectrex** usa o checksum MD5.
+
+## Magnavox
+
+- **Odyssey2** usa o checksum MD5.
+
+## Mattell
```

Comment:
Should this be with one "l" or two "l"'s? The company name is "Mattel".

Reply:
@wescopeland The correct one is MATTEL (https://en.wikipedia.org/wiki/Mattel). I reserved these PRs just for mere translation, I was thinking on doing corrections in separate PRs. But if you authorize me I can correct in both places, any time I stumble upon an error, or inconsistency. What do you prefer?

Reply:
I am much more relaxed in docs than in RAWeb. I prefer whatever you find easiest :-)

Reply:
@wescopeland Sounds good! If you don't mind, I prefer to open a PR with those documentation corrections. That way I can earn more points in my course, by accumulating those small changes into a larger contrib.

Reply:
I understand. That should be fine.